dev/storage.h

```diff
@@ -1211,6 +1208,9 @@ namespace sqlite_orm {
             template<class Table, satisfies<is_table, Table> = true>
             sync_schema_result sync_table(const Table& table, sqlite3* db, bool preserve);
 
+            template<class Table, satisfies<is_table, Table> = true>
+            sync_schema_result _sync_normal_table(const Table& table, sqlite3* db, bool preserve);
```

Comment:
func name starting with underscore looks very odd. I think regular `sync_normal_table` is well enough